mm/kfence/core.c

```diff
@@ -213,7 +212,8 @@ static inline void for_each_canary(const struct kfence_metadata *meta, bool (*fn
 			break;
 	}
 
-	for (addr = meta->addr + size; addr < PAGE_ALIGN(meta->addr); addr++) {
+	for (addr = meta->addr + meta->size; addr < PAGE_ALIGN(meta->addr);
+	     addr++) {
```

Comment:
This should not line-break (I tried to clang-format it and it doesn't with our .clang-format). Is your clang-format-diff picking up the right .clang-format?

Reply:
I broke this one manually, as they now say 80 chars is still the preferred limit. But maybe it doesn't make much sense to break a for-loop

Reply:
... especially given another for-loop crossing the 80-col limit a couple of lines above.